### ISSUE TITLE
how Can I solve this problem 

### ISSUE THREAD

**Issue by LamnouarMohamed** [neuralmagic/sparsify#121]
Hi, 

Thank you so much for sharing this repository, how can I solve this problem in image below 
![problem ](https://user-images.githubusercontent.com/26192678/184160106-bdaee083-cd82-4648-8eb0-6de6ae64f608.png)

**Comment by bfineran** [comment]
Hi @LamnouarMohamed you can either unselect performance profiling, or `pip install deepsparse` and restart the sparsify application

**Comment by LamnouarMohamed** [comment]
Thank you so much for reply, it's already done, but not yet it's same problem

**Comment by cyprian** [comment]
I am also facing the same problem. 
I `pip installed deepsparce`, run the model analysis via web interface, and the interface says" The deepsparcse package must be installed for CPU performance" when Performance check is on.

**Comment by jeanniefinks** [comment]
Hi @LamnouarMohamed 
It appears a fix had been merged to hopefully address this issue.

In other news, the Neural Magic team has been busy working on the next generation of Sparsify which will have much-improved pathways for users to interact and get results. We are very excited to see that go live later this year. As such, we will be closing this ticket. 

We did want to invite users who are interested in being Sparsify testers and giving feedback to contact our UX lead Rob Fitzgibbon (@NMfitz). We would be happy to make it worth your time with a gift card if you do! 

Best and thank you, Jeannie / Neural Magic
